### PR TITLE
Add the option to specify modal view height

### PR DESCRIPTION
**Pull request by arturs105**

### CODE REVIEW

**Comment by willsb** [comment]
I'm not a big fan of this approach. It looks really fragile, cumbersome and it `CGRect`s. I'd rather try and use constraints instead, since those are more resilient to changes in the window size (something that happens when the user receives a call, for example)

**Comment by arturs105** [comment]
Good point, constraints probably would be better. Where do you suggest these constraints be set? In the view controller itself or somewhere else?

**Comment by amulware** [comment]
This has not changed in almost a week. What's gonna happen? @arturs105 @willsb

**Comment by arturs105** [comment]
I am going to look at this today, but probably have to wait for @willsb since he might have a better idea how to do this.

**Comment by arturs105** [comment]
So the height for modals can now be set from the view controller by setting the `PreferredContentSize`. The size can also be changed/animated from the view controller by manipulating the `Frame` and `PreferredContentSize` properties.

File: Toggl.Daneel/Presentation/TogglPresenter.cs [hunk 1]
```diff
@@ -55,9 +55,9 @@ protected virtual void ShowNestedViewController(UIViewController viewController,
         protected virtual void ShowModalCardViewController(UIViewController viewController, MvxBasePresentationAttribute attribute, MvxViewModelRequest request)
         {
             viewController.ModalPresentationStyle = UIModalPresentationStyle.Custom;
-            viewController.TransitioningDelegate = FromBottomTransitionDelegate;
```
Comment (willsb) [inline Toggl.Daneel/Presentation/TogglPresenter.cs]: Is the order change still needed?

File: Toggl.Daneel/Presentation/Transition/FromBottomTransitionDelegate.cs [hunk 2]
```diff
@@ -25,6 +25,7 @@ public IUIViewControllerAnimatedTransitioning GetAnimationControllerForDismissed
         public IUIViewControllerInteractiveTransitioning GetInteractionControllerForDismissal(IUIViewControllerAnimatedTransitioning animator)
             => swipeInteractionController.InteractionInProgress ? swipeInteractionController : null;
 
-        public void WireToViewController(UIViewController vc) => swipeInteractionController.WireToViewController(vc);
```
Comment (willsb) [inline Toggl.Daneel/Presentation/Transition/FromBottomTransitionDelegate.cs]: `git checkout develop -- Toggl.Daneel/Presentation/Transition/FromBottomTransitionDelegate.cs`

**Comment by arturs105** [comment]
Yesterday I tested only on an emulator and it worked, just tried out on an iPhone and it still works (with and without `PreferredContentSize`).

**Pull request merged**